cryptofeed/exchange/coinbase.py

```diff
@@ -26,10 +26,16 @@ class Coinbase(Feed):
 
     def __init__(self, pairs=None, channels=None, callbacks=None, **kwargs):
         super().__init__('wss://ws-feed.pro.coinbase.com', pairs=pairs, channels=channels, callbacks=callbacks, **kwargs)
+        # we only keep track of the order book if we have at least one subscribed order-book callback
+        self.keep_order_book = False
+        for cb_type in callbacks:
+            if cb_type in (L3_BOOK, L2_BOOK, BOOK_DELTA):
```

Comment:
Pretty sure L2 book doesnt need to be touched, does it? You'd only ignore L3 books and L3 deltas if L3 was enabled but there was not callback for it